index.js

```diff
@@ -57,7 +57,7 @@ function loader (mcVersion) {
   }
 
   const windowByType = new Map()
-  for (const key of Object.keys(windows)) {
+  for (const key in windows) {
```

Comment:
don't use in, in javascript it's bad
prefer for or .forEach

Reply:
sorry for that one, removed.